samples/application-conf/zCEE3.properties

```diff
@@ -0,0 +1,20 @@
+# Releng properties used by language/zCEE3.groovy
+
+#
+# Comma separated list of required build properties for zCEE3.groovy
```

Comment:
The required build properties should not be part of the `application-conf` folder.

What I noticed - all these settings actually are not really application specific, are they? They are a copy of the global configuration. I don't think, that we need to provide a sample in `application-conf`, and if we want then you missed this file specified on the `applicationPropFiles` list.

Reply:
Property files removed from the samples